### DISCUSSION TITLE
Time it takes for Liquid Soap to take over

### DISCUSSION THREAD

**Discussion by shaunzyb** [AzuraCast/AzuraCast#5220] [Q&A]
Hi

We broadcast 24hr from our studios.

We have a Liquid Soap playlist loaded up for in case we loose the connection from the studio.

This works well but we have started having issues with our studio broadband connection and sometime it drops for a couple of seconds at random time per day.

Is it possible to have a delay till LS cuts in or can we increase a buffer size perhaps, would that help?

Any thoughts appreciated!

**Comment by BusterNeece** [comment]
@shaunzyb The buffer should be configurable via the Station Profile. I believe it's classified as an "advanced" setting, so you may need to have those enabled at a system level.
